extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -115,6 +116,50 @@ export async function createAzureFunction(node?: ITreeNodeInfo): Promise<void> {
 			telemetryStep = CreateAzureFunctionStep.launchFromCommandPalette;
 
 			// prompt user for connection profile to get connection info
+			while (true) {
+				connectionInfo = await vscodeMssqlApi.promptForConnection(true);
+				if (!connectionInfo) {
+					// User cancelled
+					return;
+				}
+				telemetryStep = 'getConnectionInfo';
+				try {
+					await vscode.window.withProgress(
+						{
+							location: vscode.ProgressLocation.Notification,
+							title: constants.connectionProgressTitle,
+							cancellable: false
+						}, async (_progress, _token) => {
+							// list databases based on connection profile selected
+							connectionURI = await vscodeMssqlApi.connect(connectionInfo!);
+						}
+					);
+				} catch (e) {
+					// connection error occurred
```

Comment:
connect will show an error message if the connection fails right? Please add a comment explaining that so there isn't confusion about why we don't display anything here

And please update this comment to explain why we continue instead of returning here if an error occurs.